cadquery/cq.py

```diff
@@ -1468,6 +1468,48 @@ def parametricCurve(self, func, N=400, start=0, stop=1):
 
         return self.spline(allPoints,includeCurrent=False,makeWire=True)
 
+    def ellipseArc(self, x_radius, y_radius, angle1=360, angle2=360, rotation_angle=0.0, sense=1,
+                forConstruction=False, startAtCurrent=True, makeWire=False):
+        """Draw an elliptical arc with x and y radiuses either with start point at current point or
+        or current point being the center of the arc
+
+        :param x_radius: x radius of the ellipse (along the x-axis of plane the ellipse should lie in)
+        :param y_radius: y radius of the ellipse (along the y-axis of plane the ellipse should lie in)
+        :param angle1: start angle of arc
+        :param angle2: end angle of arc (angle2 == angle1 return closed ellipse = default)
+        :param rotation_angle: angle to rotate the created ellipse / arc
+        :param sense: clockwise (-1) or counter clockwise (1)
+        :param startAtCurrent: True: start point of arc is moved to current point; False: center of
+            arc is on current point
+        :param makeWire: convert the resulting arc edge to a wire
+        """
+
+
+        # Start building the ellipse with the current point as center
+        # to support strating as the result of center(x,y) or moveTo(x,y) do not use local coordinates
+        center = self._findFromPoint(useLocalCoords=False)
+        e = Edge.makeEllipse(x_radius, y_radius, center, Vector(0, 0, 1), angle1, angle2, sense==1)
```

Comment:
if center is in global coords the Vector(0,0,1) is NOK I think. You might want to try: `self.plane.zDir`

Reply:
yes, it is wrong. I will change and add a test case for a transformed plane